### DISCUSSION TITLE
Brackets can not online preview

### DISCUSSION THREAD

**Discussion by lemonca11** [brackets-cont/brackets#229] [Q&A]
my brackets is download on a mac os,but the editor prompt error and can not online preview

<img width="1009" alt="image" src="https://user-images.githubusercontent.com/80398419/195968622-355ac461-be64-4e39-b379-dc65b96a610f.png">

**Comment by PunyTRexArms** [comment]
The preview function in Brackets is relevant for HTML, not JS.
